### PR TITLE
Create flask cli

### PR DESCRIPTION
**Pull request by kenessajr**
Hi, @erwagasore,

I have added Pindo CLI - inituser. This command help creating users. Run inituser --help for more description on the usage on the command. inituser have been tested and fully covered.
Kindly review the `pr`.

### CODE REVIEW

File: api/app/test_app.py [hunk 1]
```diff
@@ -73,6 +74,29 @@ def test_command(self):
         result = runner.invoke(tst, cmd_options)
         assert result.exit_code == 0
 
+    def test_inituser_command(self):
+        runner = app.test_cli_runner()
+        # test anon user
+        result = runner.invoke(
+            inituser, input='kenessa\nkenessa@email.com\nkenessa\nkenessa\nsuper\n')
+        type = re.search('User type: (.+?)\n', result.output).group(1)
```
Comment (erwagasore) [inline api/app/test_app.py]: Instead of `import re` and use regex. Just check the `result.output`. Something like
```
assert 'super' in result.output
```

Comment (erwagasore) [reply api/app/test_app.py]: The more concrete test should check if the created `user` is committed to the database.
```
super = User.query.filter_by(username='kenessa').first()
assert super.is_super() == True
...
```

Comment (erwagasore) [inline api/app/test_app.py]: Also checking `exit_code` is important.

Comment (kenessajr) [reply api/app/test_app.py]: ok

File: api/app/test_app.py [hunk 2]
```diff
@@ -73,6 +74,29 @@ def test_command(self):
...
+        assert type == 'super'
+        assert result.exit_code == 0
+        assert result.exception is None
+
+        result = runner.invoke(
+            inituser, input='kenessa\nkenessa@email.com\nkenessa\nkenessa\nsuper\n')
+        type = re.search('User type: (.+?)\n', result.output).group(1)
+        assert result.exit_code == 1
```
Comment (erwagasore) [inline api/app/test_app.py]: `exit_code` should be `0`. What is the reason for having `1`

Comment (kenessajr) [reply api/app/test_app.py]: It's one because I'm creating a duplicated user

**Pull request merged**